aya/src/obj/btf/btf.rs

```diff
@@ -472,4 +815,237 @@ mod tests {
         assert_eq!(header.str_off, 0x2a5464);
         assert_eq!(header.str_len, 0x1c6410);
     }
+
+    #[test]
+    fn test_parse_btf() {
+        let data: &[u8] = &[
```

Comment:
a comment on how to generate this would help when this test inevitably breaks :P